### DISCUSSION TITLE
Cannot find name 'Transferable'. Did you mean 'Transfer'?@types/workerpool

### DISCUSSION THREAD

**Discussion by sarowarhosen03** [DefinitelyTyped/DefinitelyTyped#67030] [Issues with a @types package]
in my recent typescript project made with workerpool i face this issu , can you please fix this issu ,
at node_modules/@types/workerpool/index.d.ts:40:61 ,

`> reemit@1.0.0 build

tsc

node_modules/@types/workerpool/index.d.ts:40:61 - error TS2552: Cannot find name 'Transferable'. Did you mean 'Transfer'?

40 options?: { on?: (payload: any) => void; transfer?: Transferable[] },
~~~~~~~~~~~~

node_modules/@types/workerpool/index.d.ts:65:14
65 export class Transfer {
~~~~~~~~
'Transfer' is declared here.

node_modules/@types/workerpool/index.d.ts:67:15 - error TS2552: Cannot find name 'Transferable'. Did you mean 'Transfer'?

67 transfer: Transferable[];
~~~~~~~~~~~~

node_modules/@types/workerpool/index.d.ts:65:14
65 export class Transfer {
~~~~~~~~
'Transfer' is declared here.

node_modules/@types/workerpool/index.d.ts:73:41 - error TS2552: Cannot find name 'Transferable'. Did you mean 'Transfer'?

73 constructor(message: any, transfer: Transferable[]);
~~~~~~~~~~~~

node_modules/@types/workerpool/index.d.ts:65:14
65 export class Transfer {
~~~~~~~~
'Transfer' is declared her`
![Screenshot from 2023-10-12 00-29-28](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/98280301/50f89d21-
![Screenshot from 2023-10-12 00-29-42](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/98280301/4ad1b279-3ec8-40cf-a6a9-4e7c8a9a4418)
5f78-4a22-a1f1-1daab194d4fe)

[a comment by an automated account is not shown]

**Comment by sarowarhosen03** [comment]
@types/workerpool/index.d.ts Here is the file

**Comment by Semigradsky** [comment]
@sarowarhosen03 update your TS, you use very old version.

Or add `Transferable` to global himself:
```ts
type Transferable = OffscreenCanvas | ImageBitmap | MessagePort | ReadableStream | WritableStream | TransformStream | VideoFrame | ArrayBuffer;
```

**Reply by sarowarhosen03** [reply]
Thank you but I am solved it by adding  `skipLibChek=true` in my tsconfig.json
